iree/turbine/kernel/wave/codegen.py

```diff
@@ -1321,7 +1321,6 @@ def handle_reshape(emitter: WaveEmitter, node: fx.Node):
         raise ValidationError("Malformed arguments") from e
     custom = get_custom(node)
     innermost_dim = custom.type.symbolic_shape[-1]
-    offset = custom.expanded_dims[innermost_dim]
```

Comment:
Will do!

the reason I moved lower is because the offset is only being used by Reshapes that does "extract". Since for combining case, we don't really use/depend on the expanded_dims at all, I moved it away from there S.T it doesn't break "combiner" reshapes that we generate post expansion.

Reply:
done

Reply:
thanks!